lib/api/twitter/data/tweet.dart

```diff
@@ -54,6 +54,8 @@ class Tweet {
 
   bool get emptyText => displayTextRange[1] == 0;
 
+  String get shareUrl => "https://twitter.com/${user.screenName}/status/$idStr";
```

Comment:
I would prefer to have the shareUrl inside the `TweetShareButton`, since it shouldn't be part of tweet model

Reply:
Done: https://github.com/robertodoering/harpy/pull/146/commits/56710ed4fe5c6b0b2e6a61fdefc06323e40c8655